spec/calendar.html

```diff
@@ -844,7 +850,8 @@ <h1>ResolveISOMonth ( _fields_ )</h1>
         1. If _monthCode_ is *undefined*, then
           1. If _month_ is *undefined*, throw a *TypeError* exception.
           1. Return ℝ(_month_).
-        1. Assert: Type(_monthCode_) is String.
+        1. Assert: _monthCode_ is a String.
+        1. NOTE: The ISO 8601 calendar does not include leap months.
         1. If the length of _monthCode_ is not 3, throw a *RangeError* exception.
         1. If the first code unit of _monthCode_ is not 0x004D (LATIN CAPITAL LETTER M), throw a *RangeError* exception.
         1. Let _monthCodeDigits_ be the substring of _monthCode_ from 1.
```

Comment:
I didn't notice before, but there's a type issue in the steps after this... `_monthCodeNumber_` is the output of ToIntegerOrInfinity and is therefore not a Number but rather an extended mathematical value, which cannot be passed to SameValue.

Reply:
Fixed in #2580.